lib/matplotlib/backend_bases.py

```diff
@@ -2657,6 +2657,14 @@ def mouse_move(self, event):
                             if data_str is not None:
                                 s = s + ' ' + data_str
 
+                if self.mode == 'zoom rect' and self._xypress:
+                    lastx, lasty, a, ind, view = self._xypress[0]
+                    trans = event.inaxes.transData.inverted()
+                    last_xdata, last_ydata = trans.transform_point((lastx, lasty))
+                    dx = event.inaxes.format_xdata(event.xdata - last_xdata)
+                    dy = event.inaxes.format_ydata(event.ydata - last_ydata)
+                    s += ' (Δx={}\tΔy={})'.format(dx, dy)
```

Comment:
Please stick to ascii in the source file; you can use a named entity (`"\N{GREEK CAPITAL LETTER DELTA}"`) instead.